src/stan/variational/advi.hpp

```diff
@@ -374,13 +553,29 @@ namespace stan {
       BaseRNG& rng_;
       int n_monte_carlo_grad_;
       int n_monte_carlo_elbo_;
-      double eta_adagrad_;
       int eval_elbo_;
       int n_posterior_samples_;
       std::ostream* print_stream_;
       std::ostream* out_stream_;
       std::ostream* diag_stream_;
 
+      double circ_buff_median_(const boost::circular_buffer<double>& cb) const {
```

Comment:
1. where is it duplicated?
2. let's move these functions to stan-math in a separate pull request after merging.